WordPress/Classes/ViewRelated/Stats/Traffic/Chart/StatsTrafficBarChartCell.swift

```diff
@@ -333,11 +306,42 @@ struct StatsTrafficBarChartTabData: FilterTabBarItem, Equatable {
         return self.tabTitle
     }
 
+    var attributedTitle: NSAttributedString? {
+        let attributedTitle = NSMutableAttributedString(string: tabTitle)
+        attributedTitle.addAttributes([.font: TextStyle.footnote.uiFont],
+                                      range: NSMakeRange(0, attributedTitle.string.count))
```

Comment:
nitpick: I see both `attributedString.string.count` and `attributedTitle.length` (used below). I think the latter is best and would be what `addAttributes(:range:)` expects. It won't cause any issues here since we don't include any odd characters like emoji.